dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/IEC61400/Parameters/QControlParameters2020.mo

```diff
@@ -12,7 +12,7 @@ within Dynawo.Electrical.Controls.IEC.IEC61400.Parameters;
 * This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
 */
 
-record QControlParameters
+record QControlParameters2020
```

Comment:
Also, could you add this change to the updateXML file? you can put it in update1.7.5.py

Reply:
I've put it in update1.7.0.6.py considering what was done for the previous tickets is it ok ?